### PR TITLE
test(e2e): k6 stake pool search

### PR DESCRIPTION
**Pull request by iadmytro**
# Context

K6 test for stake pool search endpoint

### CODE REVIEW

**Review by rhyslbw — APPROVED** [review]

File: packages/e2e/test/k6/endpoints/stake-pool/search.test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      }
+    );
+    check(searchResponse, {
+      'Initial query successfull': (r) => r.status === 200
+    });
+    const parsedPageResults = JSON.parse(searchResponse.body).pageResults;
+    stakePools = [...stakePools, ...parsedPageResults];
```
Comment (iccicci) [inline packages/e2e/test/k6/endpoints/stake-pool/search.test.js:36]:
```suggestion
    stakePools.push(...parsedPageResults);
```

is equivalent but avoids to allocate a new `Array` and _copy all the elements_ at each iteration.

**Review by iccicci — DISMISSED** [review]
LGTM!

Just a minor performances improvement suggestion.

**Review by mirceahasegan — APPROVED** [review]
Great work, @iadmytro ! 🚀

File: packages/e2e/test/k6/endpoints/stake-pool/search.test.js (hunk 1 above)
Comment (rhyslbw) [reply packages/e2e/test/k6/endpoints/stake-pool/search.test.js]: Is this ready for re-review @iadmytro?

**Review by iccicci — APPROVED** [review]

**Pull request merged**